requirements-dev.txt

```diff
@@ -1,6 +1,8 @@
 -r requirements.txt
 mypy~=1.10.0
-ruff==0.4.7
+ruff~=0.4.7
 pytest-cov~=5.0.0
 requests-mock~=1.12.1
 types-requests~=2.27.1
+tox~=4.18.1
```

Comment:
Included a bit early in the requirements (see comment in commit)

[from README.md in feat/update_project_management](https://github.com/AntaresSimulatorTeam/antares_craft/blob/feat/update_project_management/README.md):

### tox
To use [tox](https://tox.wiki/) to run unit tests in multiple python versions at the same time as linting and formatting
with ruff and typing with mypy:
1) As the dev requirements include [uv](https://docs.astral.sh/uv/) and `tox-uv` there is no need to install python 
versions, `uv` will do this for you.
2) Use `tox -p` to run the environments in parallel to save time, this will create virtual environment with the 
necessary python versions the first time you run tox.

Reply:
Also possible it slipped in by mistake since I've been using tox locally for awhile, either way including it is intentional in the other branch.

Reply:
Okay why not